certified-connectors/Connect2All On-Premises/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4024 @@
+                    "description": "Length",
+                    "format": "int32",
+                    "type": "integer"
+                  },
+                  "name": {
+                    "description": "Name",
+                    "type": "string"
+                  },
+                  "options": {
+                    "description": "Options",
+                    "properties": {
+                      "0": {
+                        "description": "0",
+                        "type": "string"
+                      },
+                      "1": {
+                        "description": "1",
+                        "type": "string"
+                      },
+                      "2": {
+                        "description": "2",
+                        "type": "string"
+                      }
+                    },
+                    "type": "object"
+                  },
+                  "relatedfieldid": {
+                    "description": "Related Field Id",
+                    "format": "int32",
+                    "type": "integer"
+                  },
+                  "relatedtableid": {
+                    "description": "Related Table Id",
+                    "format": "int32",
+                    "type": "integer"
+                  }
+                },
+                "type": "object"
+              },
+              "type": "array"
+            }
+          },
+          "400": {
+            "$ref": "#/responses/Unexpected"
+          },
+          "401": {
+            "$ref": "#/responses/Unauthorized"
+          },
+          "402": {
+            "$ref": "#/responses/InvalidLicense"
+          },
+          "404": {
+            "$ref": "#/responses/NotFound"
+          },
+          "500": {
+            "$ref": "#/responses/Unexpected"
+          }
+        },
+        "summary": "Get the fields of a table",
+        "tags": [
+          "helpers"
+        ],
+        "x-ms-visibility": "internal"
+      }
+    },
+    "/api/2.0/hooks": {
+      "x-ms-notification-content": {
+        "description": "This is the system id is that is send back",
+        "schema": {
+          "properties": {
+            "SystemId": {
+              "description": "SystemId",
+              "type": "string"
+            }
+          },
+          "type": "object"
+        }
+      }
+    },
+    "/api/2.0/hooks/create": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "description": "A record is created in Business Central",
+        "operationId": "RecordCreatedV2",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Url"
+          },
+          {
+            "$ref": "#/parameters/Company"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "required": true,
+            "schema": {
+              "properties": {
+                "CallbackUrl": {
+                  "description": "CallbackUrl",
```

Comment:
@Joerio strings in camel case are considered non-translatable. Please make sure that descriptions look like valid sentences. More information about quality strings can be found in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).